app/services/langfuse/tasks/score_monitor/score.py

```
@@ -299,6 +299,106 @@ async def send_score_alert(
+                return False
+
+            summary_sent_key = f"alerts:summary_sent:{today}"
+
+            # Check if we already sent today's summary
+            try:
+                redis_service = await get_redis_service()
+                already_sent = await redis_service.get(summary_sent_key)
+                if already_sent:
+                    logger.debug(f"Daily alert summary already sent for {today} - skipping")
+                    return False
+            except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)